### ISSUE TITLE
Documentation review

### ISSUE THREAD

**Issue by alvarolopez** [EGI-FCTF/cloud-info-provider#36]
The current README is hard to follow, we should make it more crisp, and add some proper documentation in separate files.

**Comment by gwarf** [comment]
According to [recommneded community standards](https://opensource.guide/) it's also required to add (or extract) this missing information:
- [ ] code of conduct
- [ ] Contributing
- [ ] License
- [ ] Issue or pull request template

**Comment by brucellino** [comment]
Most of the above are done in #77. I will take a look at the documentation, with virgin eyes.

**Comment by brucellino** [comment]
Added the license (apache 2.0, owned by @EGI-foundation

**Comment by gwarf** [comment]
- [ ] The License should not be onwed by @EGI-Foundation but by some like "the Authors".
- [ ] Every provider should be docuented individually and completely
